### PR TITLE
feat(seamless) chore: add `skipranges` and `replaces` logic TODOs to CSV

### PR DESCRIPTION
**Pull request by nickboldt**
### What does this PR do?

feat(seamless) chore: add skipranges and replaces logic TODOs, which we can enable when 0.1 and 1.1 are live
alternatively, we could enable this sooner but then to install 1.2 you have to FIRST install 1.1, etc.

Signed-off-by: Nick Boldt <nboldt@redhat.com>

### Screenshot/screencast of this PR
N/A

### What issues does this PR fix or reference?
N/A (or see commit message above for issue number)

### How to test this PR?
N/A

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)

- [x] [The Eclipse Contributor Agreement is valid](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-eclipse-contributor-agreement-is-valid)
- [x] [Code produced is complete](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-produced-is-complete)
- [x] [Code builds without errors](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-builds-without-errors)
- [ ] [Tests are covering the bugfix](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#tests-are-covering-the-bugfix)
- [x] [The repository devfile is up to date and works, if one exists](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-repository-devfile-is-up-to-date-and-works)
- [x] [Sections What issues does this PR fix or reference and How to test this PR completed](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#sections-what-issues-does-this-pr-fix-or-reference-and-how-to-test-this-pr-completed)
- [x] [Relevant user documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [x] [Relevant contributing documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [x] [CI/CD changes implemented, documented and communicated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#cicd-changes-implemented-documented-and-communicated)

### Reviewers

Reviewers, please comment how you tested the PR when approving it.

### CODE REVIEW

File: bundle/manifests/backstage-operator.clusterserviceversion.yaml
Comment (rm3l) [inline bundle/manifests/backstage-operator.clusterserviceversion.yaml:1]: I think the changes in this file will be lost if we run `make bundle` again.

I think you should update the base manifest in `config/manifests/bases/backstage-operator.clusterserviceversion.yaml` instead, then regenerate with `make bundle`.

(Another example of why  https://github.com/janus-idp/operator/pull/195 could be helpful ^^)

**Review by rm3l — CHANGES_REQUESTED** [review]

File: bundle/manifests/backstage-operator.clusterserviceversion.yaml
Comment (nickboldt) [reply bundle/manifests/backstage-operator.clusterserviceversion.yaml:1]: ok, added the change to `config/manifests/bases/backstage-operator.clusterserviceversion.yaml` too

File: config/manifests/bases/backstage-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -52,4 +53,6 @@ spec:
   provider:
     name: Red Hat Inc.
     url: https://www.redhat.com/
-  version: 0.0.0
+  version: 0.2.0
```
Comment (nickboldt) [inline config/manifests/bases/backstage-operator.clusterserviceversion.yaml:56]: is this OK? setting 0.0.0 here is fine but we need a specific `skiprange` and `replaces` that changes with every release.

Comment (rm3l) [reply config/manifests/bases/backstage-operator.clusterserviceversion.yaml:56]: I think this gets replaced by the `VERSION` defined in the `Makefile`. So it should be fine, I guess..

**Review by rm3l — APPROVED** [review]

**Pull request merged**